incubator/binding-openapi-asyncapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/asyncapi/internal/config/OpenapiAsyncCompositeBindingAdapter.java

```diff
@@ -119,6 +122,8 @@ private <C> BindingConfigBuilder<C> injectHttpKafkaRoutes(
                         final AsyncapiChannelView channel = AsyncapiChannelView
                             .of(asyncapi.channels, asyncapiOperation.channel);
 
+                        final boolean includeKey = pathId.reset(item).find();
```

Comment:
I think this should not depend on the name of the parameter being `id`, for example `taxiId` should be fine too.

Also, the terminology `includeKey` is describing anticipated usage of the parameter, which is leaking the abstraction a bit. Instead, lets call this `hasParam`, or perhaps even passing the parameter name(s) so we are not so dependent on it being `{id}` in the path vs `{taxiId}` for example.